CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -2276,7 +2323,11 @@ protected override void OnMouseMove(MouseEventArgs e)
                 var point = e.GetPosition(this);
                 var modifiers = e.GetModifiers();
 
-                browser.GetHost().SendMouseMoveEvent((int)point.X, (int)point.Y, false, modifiers);
+                if (!mouseTeleport.IsInsideOriginalRect((int)point.X, (int)point.Y))
```

Comment:
Under what instance do we want to ignore the mouse move?

These methods should simple call `GetAdjustedMouseCoords`, no logic should be in any of these methods.

Reply:
I moved the logic to the class.